app/api/users.py

```diff
@@ -215,3 +219,19 @@ class UserRelationship(ResourceRelationship):
     schema = UserSchema
     data_layer = {'session': db.session,
                   'model': User}
+
+
+@user_misc_routes.route('/users/checkEmail', methods=['POST'])
+def is_email_available():
+    email = request.json.get('email', None)
+    if email:
+        if get_count(db.session.query(User).filter_by(email=email)):
+            return jsonify(
+                result="False"
+            )
+        else:
+            return jsonify(
+                result="True"
+            )
```

Comment:
Declare a schema and use that. See how the `ChargesLayer` is used without any model(In app/api/orders.py). Similarly you can create a `UserLayer`.

Reply:
It seems like this will be an overkill. You can leave it as it is. @mayank8318

Reply:
That seems like an overkill for this IMO